### DISCUSSION TITLE
Auth guard [api]

### DISCUSSION THREAD

**Discussion by HazemMohamed252** [statamic/cms#5905] [Q&A] [answered]
How to define auth guard [api] to be able to use Rest APIs and where to put it ? Any help please ?

**Comment by jasonvarga** [comment, marked as answer]
Put it in your `config/auth.php` file.

https://github.com/statamic/statamic/blob/1733fea5ff4855d72ca2a163d155f9a23d6f833a/config/auth.php#L38-L43

https://laravel.com/docs/9.x/authentication

**Reply by HazemMohamed252** [reply]
Many thanks
